persistence/build.gradle

```diff
@@ -11,6 +11,7 @@ dependencies {
     implementation project(':data')
     implementation project(':core-models')
     implementation project(':device')
+    implementation project(':domain')
```

Comment:
this breaks our architectural principles – consider putting this provider NetworkModeProvider in device module